plugins/inputs/neptune_apex/README.md

```diff
@@ -0,0 +1,145 @@
+  #response_timeout = "5s"
+
+```
+
+### Metrics
+
+The [Neptune Apex](https://www.neptunesystems.com/) controller family allows an aquarium hobbyist to monitor and control
+their tanks based on various probes. The data is taken directly from the /cgi-bin/status.xml at the interval specified 
+in the telegraf.conf configuration file. 
+
+No manipulation is done on any of the fields to ensure future changes to the status.xml do not introduce conversion bugs
+to this plugin. When reasonable and predictable, some tags are derived to make graphing easier and without front-end 
+programming. These tags are clearly marked in the list below and should be considered a convenience rather than authoritative.
+
+- neptune_apex (All metrics have this measurement name)
+  - tags:
+    - host (mandatory, string) is the host on which telegraf runs.
+    - hostname (mandatory, string) contains the hostname of the apex device. This can be used to differentiate between 
+    different units. By using the hostname instead of the serial number, replacements units won't disturb graphs.
+    - type (mandatory, string) maps to the different types of data. Values can be "controller" (The Apex controller 
+    itself), "probe" for the different input probes, or "output" for any physical or virtual outputs. The Watt and Amp 
+    probes attached to the physical 120V outlets are aggregated under the output type.
+    - probe_type (optional, string) contains the probe type as reported by the Apex.
+    - name (optional, string) contains the name of the probe or output.
+    - output_id (optional, string) represents the internal unique output ID. This is different from the device_id.
+    - device_id (optional, string) maps to either the aquabus address or the internal reference.
+    - output_type (optional, string) categorizes the output into different categories. This tag is DERIVED from the 
+    device_id. Possible values are: "variable" for the 0-10V signal ports, "outlet" for physical 120V sockets, "alert" 
+    for alarms (email, sound), "virtual" for user-defined outputs, and "unknown" for everything else.
+  - fields:
+    - value (float, various unit) represents the probe reading.
+    - state (string) represents the output state as defined by the Apex. Examples include "AOF" for Auto (OFF), "TBL" 
+    for operating according to a table, and "PF*" for different programs.
+    - amp (float, Ampere) is the amount of current flowing through the 120V outlet.
+    - watt (float, Watt) represents the amount of energy flowing through the 120V outlet.
+    - xstatus (string) indicates the xstatus of an outlet. Found on wireless Vortech devices.
+    - hardware (string, controller hardware version)
+    - software (string, software version)
+    - power_failed (string, date time) when the controller last lost power.
+    - power_restored (string, date time) when the controller last powered on.
```

Comment:
Usually we recommend to store times as nanosecond since epoch, but have been extremely consistent on this in the past, these are more efficient to store in the database.  With our new query language "flux" it is possible to format a time in a human readable format in the query, although currently with InfluxQL you would need to do it outside the query language.

Reply:
Ok, no problem.
FYI, grafana supports showing this as time if we divide by 1,000 and set its type to "Time". That was my use case and this workaround works.